Example code build fails with IDF 5.5 due to missing type definition

### Checklist

- [x] Checked the issue tracker for similar issues to ensure this is not a duplicate
- [x] Read the documentation to confirm the issue is not addressed there and your configuration is set correctly
- [x] Tested with the latest version to ensure the issue hasn't been fixed

### How often does this bug occurs?

always

### Expected behavior

The example code esp_video\examples\simple_video_server should build with errors

### Actual behavior (suspected bug)

While compiling an example code (esp_video\examples\simple_video_server) I stumbled upon a compilation issue:

In file included from C:/Espressif/frameworks/esp-idf-v5.5/my_camera/esp_video/examples/simple_video_server/managed_components/espressif__esp_wifi_remote/idf_v5.5/esp_eap_client_remote_weak.c:7:
C:/Espressif/frameworks/esp-idf-v5.5/my_camera/esp_video/examples/simple_video_server/managed_components/espressif__esp_wifi_remote/idf_v5.5/include/esp_eap_client_remote_api.h:37:49: error: unknown type name 'esp_eap_method_t'
   37 | esp_err_t esp_eap_client_remote_set_eap_methods(esp_eap_method_t methods);
      |                                                 ^~~~~~~~~~~~~~~~
C:/Espressif/frameworks/esp-idf-v5.5/my_camera/esp_video/examples/simple_video_server/managed_components/espressif__esp_wifi_remote/idf_v5.5/esp_eap_client_remote_weak.c:130:54: error: unknown type name 'esp_eap_method_t'
  130 | WEAK esp_err_t esp_eap_client_remote_set_eap_methods(esp_eap_method_t methods)
      |                                                      ^~~~~~~~~~~~~~~~
[45/1181] Generating ../../partition_table/partition-table.bin
Partition table binary generated. Contents:

I found a similar issue with a claim that a fix was merged to master. However when I configure my example to fetch this component from main branch using:
+++ b/esp_video/examples/simple_video_server/main/idf_component.yml
@@ -7,11 +7,13 @@ dependencies:
   protocol_examples_common:
     path: ${IDF_PATH}/examples/common_components/protocol_examples_common
   espressif/mdns: "^1.0.3"
-  esp_wifi_remote:
-    version: "*"
+  espressif/esp_wifi_remote:
+    git: https://github.com/espressif/esp-wifi-remote.git
+    path: "components/esp_wifi_remote"
+    version: "main"
     rules:
       - if: "target in [esp32p4]"
   espressif/cjson:
     version: "^1.7.15"
     rules:
-      - if: "idf_version >= 6.0"
\ No newline at end of file
+      - if: "idf_version >= 6.0"

It still fails.

### Error logs or terminal output

```shell

```

### Steps to reproduce the behavior

Clone example code esp_video\examples\simple_video_server and build using espressif console and build.py build command

### Project release version

ESP-IDF 5.5

### System architecture

Intel/AMD 64-bit (modern PC, older Mac)

### Operating system

Windows

### Operating system version

Windows 11 PRO

### Shell

CMD

### Additional context

_No response_

--- a/components/esp_wifi_remote/.cz.yaml
+++ b/components/esp_wifi_remote/.cz.yaml
@@ -3,6 +3,6 @@ commitizen:
   bump_message: 'bump(wifi_remote): $current_version -> $new_version'
   pre_bump_hooks: python ../../ci/changelog.py esp_wifi_remote
   tag_format: wifi_remote-v$version
-  version: 1.6.3
+  version: 1.6.4
   version_files:
   - idf_component.yml

--- a/components/esp_wifi_remote/CHANGELOG.md
+++ b/components/esp_wifi_remote/CHANGELOG.md
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.6.4](https://github.com/espressif/esp-wifi-remote/commits/wifi_remote-v1.6.4)
+
+### Bug Fixes
+
+- Regenerate v6.0 injected wifi types header ([ae98a2f](https://github.com/espressif/esp-wifi-remote/commit/ae98a2f))
+- Regenerate v5.3 Kconfig for roaming-app select ([f9db751](https://github.com/espressif/esp-wifi-remote/commit/f9db751))
+- Revert esp-hosted restriction (~v3.0) ([5811ce4](https://github.com/espressif/esp-wifi-remote/commit/5811ce4))
+
+### Updated
+
+- ci: use esp_hosted wifi/sta/cp example (slave dropped in v3.x) ([a76c0da](https://github.com/espressif/esp-wifi-remote/commit/a76c0da))
+
 ## [1.6.3](https://github.com/espressif/esp-wifi-remote/commits/wifi_remote-v1.6.3)
 
 ### Bug Fixes

--- a/components/esp_wifi_remote/idf_component.yml
+++ b/components/esp_wifi_remote/idf_component.yml
@@ -1,4 +1,4 @@
-version: 1.6.3
+version: 1.6.4
 url: https://github.com/espressif/esp-wifi-remote
 description: Utility wrapper for esp_wifi functionality on remote targets
 dependencies:

--- a/components/esp_wifi_remote/include/esp_wifi_remote_version.h
+++ b/components/esp_wifi_remote/include/esp_wifi_remote_version.h
@@ -15,7 +15,7 @@
  /** Minor version number (x.X.x) */
  #define ESP_WIFI_REMOTE_VERSION_MINOR   6
  /** Patch version number (x.x.X) */
- #define ESP_WIFI_REMOTE_VERSION_PATCH   3
+ #define ESP_WIFI_REMOTE_VERSION_PATCH   4
 
  /**
   * Macro to convert WIFI_REMOTE version number into an integer

